cypress/support/commands/miq_data_table_commands.js

```diff
@@ -53,3 +53,23 @@ Cypress.Commands.add('selectTableRowsByText', ({ textArray = [] }) => {
     });
   });
 });
+
+/**
+ * Command to click on a table row that contains the specified text.
+ * If columnIndex is provided, it will only look for the text in that specific column.
+ *
+ * @param {Object} params - Parameters for the command
+ * @param {string} params.text - Text to find in the table row
+ * @param {number} [params.columnIndex] - Optional index of the column to search in (0-based)
+ */
+Cypress.Commands.add('clickTableRowByText', ({ text, columnIndex }) => {
+  if (!text) {
+    cy.logAndThrowError('text parameter is required');
+  }
+
+  if (columnIndex || columnIndex === 0) {
+    cy.contains(`.miq-data-table table tbody tr td:nth-child(${columnIndex + 1})`, text).click();
+  } else {
+    cy.contains('.miq-data-table table tbody tr td', text).click();
```

Comment:
👍 It's good to be case sensitive here and expect exact text since we'll be creating the objects with the right text so it should be easy to ensure we match it correctly.

It will be interesting to see if this miq-data-table structure holds for all pages containing the data table.  I hope so but we'll see as we do more pages.